decidim-age_action_authorization/lib/decidim/age_action_authorization/authorizer.rb

```diff
@@ -7,11 +7,18 @@ def missing_fields
       end
 
       def unmatched_fields
```

Comment:
As we said, move this feature into a module in the DIBA main app and use metaprogramming to override the AgeActionAuthorization.